content/posts/February-2020.md

```diff
@@ -84,3 +84,28 @@ Following bugs were fixed:
 [(#715)]: https://github.com/packit-service/packit/pull/715
 [(#406)]: https://github.com/packit-service/packit-service/pull/406
 [(#420)]: https://github.com/packit-service/packit-service/pull/420
+
+## Week 9
+
+Dist-git patches are applied with `-p1` [(#730)]. Packit specfile code was improved [(#736)], [(#734)].
+Fedora-32 was added to 'fedora-all' and 'fedora-development' aliases [(#731)].
+Documentation was improved by adding links to user configuration [(#726)].
+
+Packit-service was adjusted, so we don't need metadata for copr build [(#431)], ansible deployment playbooks
+were adjusted, to use same playbooks in zuul and localy [(#429)], handlers were moved to separate module[(#426)]. 
+Build-helper classes were refactored [(#424)].
```

Comment:
We can maybe merge the sentences about refactoring.

Reply:
fixed